engine/transaction/record/record_outline_test.go

```diff
@@ -168,6 +160,30 @@ func TestRecordOutlineOpReturn(t *testing.T) {
 }
 
 func TestRecordOutlineOpReturnErrorCases(t *testing.T) {
+	givenUnsignedTX := fixtures.GivenTX().WithT(t).
+		WithoutSigning().
+		WithInput(1).
```

Comment:
It is not immutable - because it's hard to `deep copy` `go-sdk` stuff.
In the future - instead of defining a "blueprint" of some common transaction we can provide a function for that:

```golang
func StdTX(t testing.TB) GivenTXSpec{
	return GivenTX(t).WithInput(10).WithP2PKHOutput(1)
}

func SomeTest(t testing.TB) GivenTXSpec{
	givenTX := StdTX(t).WithOPReturn("hello world")
	
	givenTX2 := StdTX(t).WithP2PKHOutput(1)
}
```